### ISSUE TITLE
keypoint info

### ISSUE THREAD

**Issue by yonger001** [damo-cv/TransReID#42]
Hi, the num of images in keypoint_test.txt is 12080 (in this txt, 2 images have the same name), and there are 38632 images in keypoint_train.txt, so, there are 50712 in total. 
But, the dataset of image_train has 37778 images, 1678 in query, and 11579 in gallery, 51035 in total;
In summary, 50712 keypoint info VS 51035 images, How to deal with the 323 images in TransReID??? Refer to the veri.py under datasets folder, it seems that there is no special operation for this case.
Looking forward to your reply, thank you very much!

**Comment by michuanhaohao** [comment]
These 323 images are not used to train the model. Therefore, the number of used training images is less than the number of the images in the training set.

**Comment by michuanhaohao** [comment]
![image](https://user-images.githubusercontent.com/23547900/145544745-cbe55875-88fd-4741-b030-48c64f8e1a38.png)
I have checked the log. The train set missed 63 images. The test set miss only one images, which does not influence the performance.

- Used: 37715+1677+11578=50970
-  In total: 37778+1678+11579=51035

**Comment by yonger001** [comment]
Thank u for ur detailed reply !
The other question about VehicleID's view label. There are 2 viewpoints for VehicleID, so, why the view_id in vehicleid.py is always set to 1 ? 
Looking forward to your reply, thank you very much!

**Comment by yonger001** [comment]
Thank u for ur detailed reply !
The other question about VehicleID's view label. There are 2 viewpoints for VehicleID, so, why the view_id in vehicleid.py is always set to 1 ? 
Looking forward to your reply, thank you very much!
